src/utils/validation.js

```diff
@@ -0,0 +1,21 @@
+const { getErrorWithStatus } = require('./getError');
+
+const getValidId = (id, errorMessage) => {
+  const numberId = Number(id);
+
+  if (isNaN(numberId)) {
```

Comment:
Consider checking if `numberId` is a positive integer to ensure that IDs are valid database keys, as negative or zero values might not be valid IDs.